### ISSUE TITLE
lpc prompt has changed

### ISSUE THREAD

**Issue by bje-** [apple/cups#5946]
When running `lpc`, a newline has been added to the prompt. In the example below, `stat` is my input:

```
lpc> 
stat
  [stat output]
lpc>
```

**Comment by michaelrsweet** [comment]
@bje- What version of CUPS are you using? And what OS?

**Comment by bje-** [comment]
I am running CUPS on Ubuntu 20.04.

```
$ dpkg -S $(which lpc)
cups-bsd: /usr/sbin/lpc

$ dpkg -l cups-bsd | awk '/ii/ {print $3}'
2.3.1-9ubuntu1.1
```

**Comment by michaelrsweet** [comment]
Fixed in master:

[master 9d4e1ef79] Fix lpc prompt (Issue #5946)

and in OpenPrinting CUPS master:

[master e6be10920] Fix lpc prompt (Apple #5946)
